docs/references/architecture/adr-113-mempool-lanes.md

```diff
@@ -300,44 +300,49 @@ This section describes the architectural changes needed to implement an MVP of l
 mempool. The following is a summary of the key design decisions:
 - [[Lanes definition](#lanes-definition)] The list of lanes and their corresponding priorities
   will be hardcoded in the application logic.
+- [[Initialization](#initialization)] How the application configures lanes on CometBFT.
 - [[Internal data structures](#internal-data-structures)] There will be one concurrent list (CList) data structure per
   lane.
 - [[Configuration](#configuration)] All lanes will share the same mempool configuration.
 - [[Adding transactions to the mempool](#adding-transactions-to-the-mempool)] When validating a transaction via CheckTx, the
   application will optionally return a lane for the transaction.
-- [[Reaping transactions for creating blocks](#reaping-transactions-for-creating-blocks)] Transactions will be reaped from higher-priority
-  lanes first, preserving intra-lane FIFO ordering.
-- [[Transaction dissemination](#transaction-dissemination)] We will continue to use the current P2P channel for
-   disseminating transactions, and we will implement in the mempool the logic for selecting the
-   order in which to send transactions.
+- [[Transaction dissemination](#transaction-dissemination)] We will continue to use the current P2P
+   channel for disseminating transactions, and we will implement in the mempool the logic for
+   selecting the order in which to send transactions.
+- [[Reaping transactions for creating blocks](#reaping-transactions-for-creating-blocks)]
+  Transactions will be reaped from higher-priority lanes first, preserving intra-lane FIFO ordering.
+- [[Prioritization logic](#prioritization-logic)] For disseminating and reaping transactions, the
+   scheduling algorithm should be prevent starvation of low-priority lanes.
 
 ### Lanes definition
 
 The list of lanes and their corresponding priorities will be hardcoded in the application logic. A
 priority is a value of type `uint32`, with 0 being a reserved value (see below). The application
-also needs to define which of the lanes it defines is the default lane. 
+also needs to define which of the lanes in the list it defines is the **default lane**. 
 
 To obtain the lane information from the application, we need to extend the ABCI `Info` response to
 include the following fields. These fields need to be filled by the application only in case it
 wants to implement lanes.
 ```protobuf
 message InfoResponse {
   ...
-  repeated uint32 lanes = 6;
-  uint32 default_lane = 7;
```

Comment:
Is it correct to set the default lane to 7 if it is not in the list of lanes?

Reply:
`6` and `7` are not default values. They are the field number. Currently `InfoResponse` has 5 fields.